### DISCUSSION TITLE
DEV SESSION [TUESDAY] 2 JULY -- 2030 UTC+8/1430 CEST/1330 BST/ 0830 ET

### DISCUSSION THREAD

**Discussion by jackiekazil** [mesa/mesa#2156] [General]
Where: [via Matrix](https://matrix.to/#/#project-mesa:matrix.org)

Previous meeting: https://github.com/projectmesa/mesa/discussions/2145

**Proposed Agenda**

- General updates
  - ???
- GSoC project updates 
- PR discussions:
  - ???
- Other discussions
 - Mesa Geo updates (@tpike3)
 - Documentation for the deploy process, milestone labels 

cc @tpike3, @EwoutH, @Corvince , @wang-boyu , @rht, @quaquel, @adamamer20, @harshmahesheka

**Comment by jackiekazil** [comment]
Notes: 

* GSOC updates
* Tom - Submitted an MR for Mesa-Geo. -- https://github.com/projectmesa/mesa-geo/pull/211
* rht - waiting for PR to remove the flat namespace. It is the commit that needs to happen.  
PR that needs review -- https://github.com/projectmesa/mesa/pull/2091
* Jackie is leading JOSS paper for general Mesa paper; Boyu has an overleaf project to write Simulation paper and will add Jackie.

**Comment by rht** [comment]
cc: @Chan-Dong-Jun
